hono/test/utils/global-teardown.ts

```diff
@@ -0,0 +1,5 @@
+import { stop } from './db'
+
+export default async function () {
```

Comment:
What is the purpose of this function?

Reply:
This file acts as a "beforeAll" for all test cases. The purpose of this particular file is to start the database for use by all other test files that can run in a concurrent manner. Essentially this saves us from setting up and tearing down the db on each test case file.